demos/src/pa11y.mustache

```diff
@@ -787,7 +787,7 @@
 </div>
 
 <div class="demo-container demo-container--inverse">
-	<div class="o-teaser o-teaser--large o-teaser--inverse" data-o-component="o-teaser">
```

Comment:
does this replace the non-live one? is that a problem, i don't know? what do they look like

Reply:
non live is above this.

The non-live inverse was included in pa11y demo twice, probably one was meant to be inverse+live but we forgot to add the live modifier

Reply:
ah!!